LoginPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import LoginForm from './LoginForm';
+import './styles.css';
+
+function LoginPage() {
+  return (
+    <div className="login-page">
+      <div className="login-image"></div>
```

Comment:
Consider extracting the text strings into a separate constants file. This would help in managing them in one place, especially useful for localization or if the same string is used in multiple places.